app/src/main/java/org/oppia/android/app/databinding/ConstraintLayoutAdapters.java

```diff
@@ -22,19 +26,17 @@ public static void setConstraintEndToEndOf(
     constraintSet.applyTo(constraintLayout);
   }
 
-  /** Binding adapter for setting/removing a [layout_constraintStartToStartOf] to a view */
-  @BindingAdapter("app:constraintStart_toStartOfParent")
-  public static void setConstraintStart(
-      @NonNull View view, boolean setStartConstraint
+  /**
+   * Binding adapter for setting horizontal_bias to a view
+   */
+  @BindingAdapter("app:layout_constraintHorizontal_bias")
+  public static void setHorizontalBias(
+      @NonNull View view, float value
   ) {
```

Comment:
```suggestion
  public static void setHorizontalBias(@NonNull View view, float value) {
```

This can fit on one line.

Reply:
Done